applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -64,14 +64,25 @@ init() ->
 -spec allowed_methods() -> http_methods().
 -spec allowed_methods(path_token()) -> http_methods().
 -spec allowed_methods(path_token(), path_token()) -> http_methods().
-allowed_methods() ->
-    [?HTTP_GET, ?HTTP_PUT].
-allowed_methods(_) ->
-    [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
-allowed_methods(_, <<"details">>) ->
-    [?HTTP_GET];
-allowed_methods(_, <<"mute">>) ->
-    [?HTTP_POST].
+-spec allowed_methods(path_token(), path_token(), path_token()) -> http_methods().
+
+allowed_methods() -> [?HTTP_GET, ?HTTP_PUT].
+
+allowed_methods(_) -> [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
+
+allowed_methods(_, <<"details">>) -> [?HTTP_GET];
+
+allowed_methods(_, <<"lock">>) -> [?HTTP_POST];
```

Comment:
@fenollp What is the difference with POST? Also, I can understand mute and deaf states, but what to do with kick? It's not a state, it's an action (from any point of view). To put it broader: how should we denote actions and function calls in terms of HTTP API, once and for all?